travvy/src/components/Home.vue

```diff
@@ -154,38 +154,22 @@
       } catch (error) {
          console.log(error)
       }
-        },
-
-    reverseMessage: function () {
-      this.message = this.message.split('').reverse().join('')
-  },
-  
-  //This method gets all the locations (id,city,country) from the Location table
+    },
+    async nextDest(destination) {
+      //if the nextDestination button is selected, populate the drop down value with the city
+      let element = document.getElementById("Destination");
+      element.value = destination;
+    },
+    //This method gets all the locations (id,city,country) from the Location table
```

Comment:
I am glad this will allow users to click on the destination and get the information - I think this is a key feature for sprint 4. Thanks for implementing!